surveys/urls.py

```diff
@@ -9,6 +9,7 @@
 from django.urls import path
 from django.utils.translation import ugettext_lazy as _
 from django.contrib.auth.decorators import login_required
+from django.contrib.admin.views.decorators import staff_member_required
```

Comment:
Right now `staff_member_required` will redirect you to the Django admin login interface instead of our custom login interface. Could we try using the `login_url` param? See: https://stackoverflow.com/a/42038839

Reply:
Great point!